db/src/db.rs

```diff
@@ -2778,4 +2779,97 @@ mod tests {
         // Run a basic test as a sanity check.
         run_test_add(TestConn::with_sqlite(sqlite));
     }
+
+    #[test]
+    fn test_timeline_move() {
+        let mut conn = TestConn::default();
+
+        assert_transact!(conn, r#"[
```

Comment:
You can make this pass by manually allocating in the `:db.part/user` partition, like:
```rust
        assert_eq!((65536..65539),
                   conn.partition_map.allocate_entids(":db.part/user", 3));
```
(Well, pass up until the actual broken part.  See below.)